source/linear-algebra/source/03-AT/03.ptx

```diff
@@ -243,10 +244,13 @@ Which of these subspaces of <m>\IR^3</m> describes the set of all vectors that a
     <statement>
         <p>
 Let <m>T: V \rightarrow W</m> be a linear transformation.
-The <term>image</term> of <m>T</m> is an important subspace of <m>W</m> defined by
+The <term>image</term><idx>image</idx> of <m>T</m>
+is an important subspace of <m>W</m> defined by
 <me>
 \Im T = \left\{ \vec{w} \in W\ \big|\ \text{there is some }\vec v\in V \text{ with } T(\vec{v})=\vec{w}\right\}
 </me>
+Note that, if the transformation <m>T</m> is represented by the matrix <m>A</m>,
```

Comment:
I don't like that this statement spoils activity 3.3.10. Can we move this into the observation or fact that follows 3.3.10?

Reply:
I moved this to the end of Observation 3.3.11